app/uk/gov/hmrc/helptosavefrontend/controllers/UpdateEmailAddressController.scala

```diff
@@ -72,4 +85,84 @@ class UpdateEmailAddressController @Inject() (val sessionCacheConnector:  Sessio
       }
     }
   } (redirectOnLoginURL = FrontendAppConfig.checkEligibilityUrl)
+
+  def emailVerified(emailVerificationParams: String): Action[AnyContent] = authorisedForHtsWithInfo { implicit request ⇒ implicit htsContext ⇒
+    EmailVerificationParams.decode(emailVerificationParams) match {
+
+      case Failure(e) ⇒
+        logger.warn(s"Could not decode email verification parameters: ${e.getMessage}", e)
+        Ok(views.html.register.email_verify_error(BadContinueURL))
```

Comment:
is this view part of `register` package ?

Reply:
for now yes - if we want to change the structure of the frontend I reckon our frontend dev can handle that